app/wizards/schools/register_ect_wizard/ect.rb

```diff
@@ -89,6 +89,10 @@ def appropriate_body
       def lead_provider
         LeadProvider.find(lead_provider_id) if provider_led?
       end
+
+      def trs_full_name
+        [trs_first_name, trs_last_name].join(" ").strip
+      end
```

Comment:
Could we use the `Teachers::Name` service here? I know we don't have a `Teacher` at this point but the attributes have the same name so we could pass the `ECT` instance as the `teacher:` param perhaps?
Just thinking it would save duplicating building the full name?

Reply:
@tonyheadford thanks for the pointer. Wasn't aware of that service class.

Reply:
There is a `full_name_in_trs` method on that service that doesn't use corrected_name @ltello